web_external/Datasets/CreateDatasetView.js

```diff
@@ -179,8 +160,8 @@ const CreateDatasetView = View.extend({
         }
     },
 
-    uploadFinished: function (files) {
-        this.uploadedZipFiles = _.pluck(files.files, 'name');
+    uploadFinished: function (info) {
+        this.uploadedFiles = info.files;
```

Comment:
I'm seeing behavior where the File object in `info.files` loses its `id` attribute. I think it's safer to explicitly save the attributes we need, like:

````
this.uploadedFiles = _.map(info.files, (file) => ({id: file.id, name: file.name}));
````